e2e/test/RegistryManagerE2ETests.cs

```diff
@@ -104,5 +107,49 @@ public async Task RegistryManager_AddDeviceWithProxy()
                 await registryManager.AddDeviceAsync(device).ConfigureAwait(false);
             }
         }
+
+        [TestMethod]
+        public async Task RegistryManager_Query_Works()
+        {
+            // arrange
+            using var registryManager = RegistryManager.CreateFromConnectionString(Configuration.IoTHub.ConnectionString);
+            string deviceId = $"{_devicePrefix}{Guid.NewGuid()}";
+
+            try
+            {
+                Device device = await registryManager
+                    .AddDeviceAsync(new Device(deviceId))
+                    .ConfigureAwait(false);
+
+                // act
+
```

Comment:
extra new line

Reply:
They are intentional.

The "// act" applies to the entire block under it, not just the next line.

So generally, when that is the case, we add an extra new line to communicate that.